### DISCUSSION TITLE
Mr. Collins's font

### DISCUSSION THREAD

**Discussion by emtothed** [Cyfrin/foundry-full-course-cu#691] [Q&A]
hey everyone

I have an irrelevant question.
Does anyone know what font is Mr. Collins using in the vscode ?

**Comment by devmousa** [comment]
I'm not sure, but after some search using the font, it seems like "Prima Sans Mono Std Roman" font

**Reply by emtothed** [reply]
hey @devmousa 
I checked it but I guess it's not the one. it's edgier and less smooth and the letters' shapes are similar but not the same. but thanks a lot for helping.
